docs/doxygen/intro.md

```diff
@@ -42,8 +42,8 @@ communications (one-sided and two-sided), collective communication,
 and remote atomic operations required for popular parallel programming models.
 Also, the initial UCX reference implementation
 is targeted to support current network technologies such as:
-+ Open Fabrics - InfiniBand (Mellanox, Qlogic, IBM), libfabrics, iWARP, RoCE
-+ Cray GEMINI \& ARIES
++ Open Fabrics - InfiniBand (Mellanox, Qlogic, IBM), iWARP, RoCE
++ Cray's uGNI \& ARIES
```

Comment:
Apology for changing this again but I just noticed this inconsistency. uGNI is Cray's driver and ARIES/GEMINI and generation of interconnects.  Line 46 we want to change to:
`Cray uGNI - GEMINI and ARIES interconnects`

Reply:
@shamisp Done